src/index.html

```diff
@@ -2,11 +2,493 @@
+          src="./images/logo.png"
+          alt="logo"
+       >
+      </a>
+
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a href="http://index.html" class="nav__link is-active">
+              Apple
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a href="http://index.html" class="nav__link">
+              Samsung
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a href="https://index.html" class="nav__link">
+              Smartphones
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a
+              href="https://index.html"
+              class="nav__link"
+              data-qa="nav-hover"
+           >Laptops & Computers
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a href="https://index.html" class="nav__link">
+              Gadgets
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a href="https://index.html" class="nav__link">
+              Tablets
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a href="https://index.html" class="nav__link">
+              Photo
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a href="https://index.html" class="nav__link">
+              Video
+            </a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+
+    <main>
+      <article class="cards-cont">
+        <div class="card-cont" data-qa="card">
+          <img
+            src="./images/imac.jpeg"
+            alt="iMac"
+            class="card-cont__img"
+        >
+
+          <h2 class="card-cont__title">
+            APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+          </h2>
+
+          <p class="card-cont__prod-code">
+            Product code: 195434
+          </p>
+
+          <div class="card-cont__review-cont">
+            <div class="stars stars--4">
+              <div class="stars__star"></div>
+
+              <div class="stars__star"></div>
+
+              <div class="stars__star"></div>
+
+              <div class="stars__star"></div>
+
+              <div class="stars__star"></div>
+
+            </div>
+
+            <p class="card-cont__review-count">
+              Reviews: 5
+            </p>
+          </div>
+
+          <div class="card-cont__prod-price">
+            <p class="card-cont__price-title">
+              Price:
+            </p>
+
+            <p class="card-cont__price-cur">
+              $2,199
+            </p>
+          </div>
+
+          <a
+            href="#"
+            class="card-cont__button-buy"
+            data-qa="card-hover"
+        >
+            BUY
+          </a>
+        </div>
+
+        <div class="card-cont">
+          <img
+            src="./images/imac.jpeg"
+            alt="iMac"
+            class="card-cont__img"
+        >
```

Comment:
Fix code style everywhere
```suggestion
          <img
            src="./images/imac.jpeg"
            alt="iMac"
            class="card-cont__img"
          >
```